src/pages/algorithms/search-algorithms/index.md

```diff
@@ -3,13 +3,18 @@ title: Search Algorithms
 ---
 ## Search Algorithms
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/algorithms/search-algorithms/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+This kind of algorithms is looking at the problem of re-arranging an array of items in asceding order. The two most clasical examples of that is the binary-search and the mergesort algorithm.
```

Comment:
Yeah, you are right that it could be more focussed but I'm going to accept it since we are trying to get through so many today. Then it can be updated later. Thanks for reviewing this.